packages/containers/psammead-timestamp-container/src/relativeTimestamp.js

```diff
@@ -0,0 +1,36 @@
+const constructTimestamp = timeDiffMins => {
+  let timeDiff;
+  let unit;
+
+  if (timeDiffMins >= 1440) {
+    timeDiff = parseInt(timeDiffMins / 1440, 10);
+    unit = timeDiff > 1 ? 'days' : 'day';
+  } else if (timeDiffMins >= 60) {
+    timeDiff = parseInt(timeDiffMins / 60, 10);
+    unit = timeDiff > 1 ? 'hours' : 'hour';
+  } else {
+    timeDiff = timeDiffMins <= 1 ? 1 : timeDiffMins;
+    unit = timeDiff === 1 ? 'minute' : 'minutes';
```

Comment:
these lines are also increasing Code Climates cognitive complexity but I like your approach so I think it's okay 👍